timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -0,0 +1,198 @@
+and start exploring it to discover interesting things about players that could help you win your next fantasy season. 
+If you aren't an NFL fan, don't worry, you will still find this tutorial helpful as it's showcasing how to 
+get started with TimescaleDB and explore a real world dataset with SQL and Python.
+
+
+1. [Create tables][]
+2. [Ingest data from CSV files][] 
+3. [Analyze NFL data][]
+   
+## Prerequisites
+
+* Python 3
+* TimescaleDB (see [installation options][install-timescale]) 
+* [Psql][psql-install] or any other PostgreSQL client (e.g. DBeaver)
+* Pandas and Matplotlib (`pip install pandas matplotlib`)
+
+## Download the dataset
+
+Download dataset here!
+
+## Create tables
+
+You will need to create four tables:
+
+* game
+  
+  Information about each game.
+* player
+  
+  Player information.
+* play
+  
+  Play information
+* tracking
+  
+  Tracking information from each play.
+
+```sql
+CREATE TABLE game (
+    gameId INT PRIMARY KEY,
+    gameDate DATE,
+    gameTimeEastern TIME,
+    homeTeamAbbr VARCHAR(3),
+    visitorTeamAbbr VARCHAR(3),
+    week INT
+);
+
+CREATE TABLE player (
+    nflId INT,
+    height TEXT,
+    weight INT,
+    birthDate DATE,
+    collegeName TEXT,
+    position TEXT,
+    displayName TEXT
+);
+
+CREATE TABLE play (
+    gameId INT,
+    playId INT,
+    playDescription TEXT,
+    quarter INT,
+    down INT,
+    yardsToGo INT,
+    possessionTeam TEXT,
+    playType TEXT,
+    yardlineSide TEXT,
+    yardlineNumber INT,
+    offenseFormation TEXT,
+    personnelO TEXT,
+    defendersInTheBox INT,
+    numberOfPassRushers INT,
+    personnelD TEXT,
+    typeDropback TEXT,
+    preSnapVisitorScore INT,
+    preSnapHomeScore INT,
+    gameClock TIME,
+    absoluteYardlineNumber INT,
+    penaltyCodes TEXT,
+    penaltyJerseyNumber TEXT,
+    passResult TEXT,
+    offensePlayResult INT,
+    playResult INT,
+    epa DOUBLE PRECISION,
+    isDefensivePI BOOLEAN
+);
+
+CREATE TABLE tracking (
+    time TIMESTAMP,
+    x DOUBLE PRECISION,
+    y DOUBLE PRECISION,
+    s DOUBLE PRECISION,
+    a DOUBLE PRECISION,
+    dis DOUBLE PRECISION,
+    o DOUBLE PRECISION,
+    dir DOUBLE PRECISION,
+    event TEXT,
+    nflId INT,
+    displayName TEXT,
+    jerseyNumber INT,
+    position TEXT,
+    frameId INT,
+    team TEXT,
+    gameId INT,
+    playId INT,
+    playDirection VARCHAR(5),
+    route TEXT
+);
+```
+
+Add indexes to the `tracking` table speed up queries:
+
+```sql
+CREATE INDEX idx_gameid ON tracking (gameid);
+CREATE INDEX idx_playerid ON tracking (player_id);
+CREATE INDEX idx_playid ON tracking (playid);
+```
+
+## Ingest data from CSV files
+
+We have three separate CSV files for game, player, and play tables. For the tracking table, you will need to
+import data from 17 CSV files (1 file for each week in the season).
+
+Let's use a Python script that utilizes psycopg2's `copy_expert` function to ingest the data:
```

Comment:
```suggestion
You can use a Python script that uses psycopg2's `copy_expert` function to ingest the data:
```

Reply:
So, dumb question here. Is it worth showing an example with `COPY` or psql `/copy`?